### DISCUSSION TITLE
ONTAP: SnapMirror dashboard stopped working

### DISCUSSION THREAD

**Discussion by Tamas92** [NetApp/harvest#2634] [Q&A] [answered]
On Sunday the ONTAP: SnapMirror dashboard stopped working. It shows 0s and No data.  Nothing was modified on NetApp and Grafana side. We have custom SnapMirror dashboards as well these are stopped too. I checked prometheus and the data is there. FUrthermore I created a new visualisation dashboard, selected the snapmirror_lag_time and it shows data for all volumes. 
It is installed on a RedHat machine.
Grafana 10.3.1
Harvest 23.11

**Comment by Hardikl** [comment]
@Tamas92 
As per your description, It seems prometheus also didn't get the data for snapmirror object in that duration.  
Could you share your logs with us. Please send them to [ng-harvest-files@netapp.com](mailto:ng-harvest-files@netapp.com).

**Reply by Tamas92** [reply]
The data is there. I can create new visualisation dashboard with the data from today or yesterday or from any dates in the past and it is working. The data is clearly visible from the Prometheus. 
Which logs do you need?

**Reply by cgrinds** [reply]
The Harvest logs - details on collecting are listed [here](https://netapp.github.io/harvest/nightly/help/log-collection/#rpm-deb-and-native-installations).

**Reply by Tamas92** [reply]
Thank you! I sent the log files.

**Comment by Tamas92** [comment, marked as answer]
There was a bug in the snapmirror dashboard which was fixed by the guys. Thanks for your support!

**Reply by cgrinds** [reply]
Thanks for raising the issue and letting us know it was resolved with 24.02
